### PR TITLE
Implement rules for CIS OCP Section 5.2

### PR DESCRIPTION
**Pull request by rhmdnd**
Now that we have a profile and control files for CIS 1.4.0, we can start
wiring up the existing rules.

This commit ports all the existing rules we were using for the CIS
OpenShift profile into the CIS 1.4.0 version.

### CODE REVIEW

File: controls/cis_ocp_1_4_0/section-5.yml [hunk 1]
```diff
@@ -46,50 +46,59 @@ controls:
...
     - id: 5.2.9
       title: Minimize the admission of containers with capabilities assigned
-      status: pending
-      rules: []
+      status: partial
+      rules:
+        - scc_drop_container_capabilities
       level: level_2
     - id: 5.2.10
       title: Minimize access to privileged Security Context Constraints
```
Comment (rhmdnd) [inline controls/cis_ocp_1_4_0/section-5.yml:104]: We have a rule floating around that we used for STIG that we can reuse here I think?

CC: @jhrozek

Comment (jhrozek) [reply controls/cis_ocp_1_4_0/section-5.yml:104]: think you could use `rbac_least_privilege` here.

File: controls/cis_ocp_1_4_0/section-5.yml [hunk 2]
```diff
@@ -46,50 +46,59 @@ controls:
...
-      rules: []
+      status: partial
+      rules:
+        - scc_limit_root_containers
       level: level_2
     - id: 5.2.7
       title: Minimize the admission of containers with the NET_RAW capability
-      status: pending
-      rules: []
+      status: partial
```
Comment (jhrozek) [inline controls/cis_ocp_1_4_0/section-5.yml:87]: I think when using partial we should add a note (there's a YAML attribute in the schema for that) or alternatively did you mean manual?

File: controls/cis_ocp_1_4_0/section-5.yml (hunk 1 above)
Comment (Vincent056) [reply controls/cis_ocp_1_4_0/section-5.yml:104]: wondering if we need to add a reference in the rule as well https://github.com/ComplianceAsCode/content/blob/master/applications/openshift/rbac/rbac_least_privilege/rule.yml

**Comment by rhmdnd** [comment]
Rebased and resolved merge conflicts.

**Review by Vincent056 — APPROVED** [review]
/lgtm

**Pull request merged**